### PR TITLE
Add IP interface loopback action to required YANG models

### PR DESCRIPTION
**Pull request by liorghub**
#### Why I did it
Add IP interface loopback action to required YANG models.
Please ignore the submodule change, it is just for creating internal build for verification.

### CODE REVIEW

**Review by dgsudharsan — CHANGES_REQUESTED** [review]
Please add UT for the yang models
Please add sampe configuration to [sample_config_db.json](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/tests/files/sample_config_db.json)

File: .gitmodules [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/Azure/sonic-sairedis
```
Comment (dgsudharsan) [inline .gitmodules:9]: Please remove this file as well as submodules from the PR

File: src/sonic-yang-models/yang-models/sonic-vlan.yang [hunk 2]
```diff
@@ -101,6 +101,13 @@ module sonic-vlan {
 						pattern "enabled|disabled";
 					}
 				}
+
+                leaf loopback_action {
+                    description "Packet action when a packet ingress and gets routed on the same IP interface";
+                    type string{
```
Comment (dgsudharsan) [inline src/sonic-yang-models/yang-models/sonic-vlan.yang:107]: Can we define a type in sonic-types and use it across the 4 files? Please check here for example

[    typedef boolean_type {](https://github.com/Azure/sonic-buildimage/blob/299e67a318ec65d1d4a060e8386477552cee29a9/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L241)

**Review by dgsudharsan — CHANGES_REQUESTED** [review]

**Comment by liorghub** [comment]
> Please add UT for the yang models Please add sampe configuration to [sample_config_db.json](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/tests/files/sample_config_db.json)

Added.

File: src/sonic-yang-models/yang-models/sonic-vlan.yang (hunk 2 above)
Comment (liorghub) [reply src/sonic-yang-models/yang-models/sonic-vlan.yang]: Done.

**Pull request closed without merging**